Formula/ethermint.rb

```diff
@@ -0,0 +1,35 @@
+class Ethermint < Formula
+  desc "Ethereum powered by Tendermint consensus"
+  homepage "https://github.com/tendermint/ethermint"
+  url "https://github.com/tendermint/ethermint/archive/v0.5.4.tar.gz"
+  sha256 "b571f316841513efa16d8c62c8eeddac3f947af591f964aa3e3b1bb55f10f732"
+
+  bottle do
+    cellar :any_skip_relocation
+  end
+
+  head do
+    url "https://github.com/tendermint/ethermint.git",
+      :branch => "develop"
+  end
+
+  depends_on "go" => :build
+  depends_on "glide" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ethermintpath = buildpath/"src/github.com/tendermint/ethermint"
+    ethermintpath.install buildpath.children
+    cd ethermintpath do
+      system "make", "get_vendor_deps"
+      system "make", "build"
+      bin.install "build/ethermint"
+    end
+  end
+
+  test do
+    ethermint_version = shell_output("#{bin}/ethermint version").split("\n").first.partition("  ")[2]
```

Comment:
Test is failing:

```
ethermint version
panic: runtime error: slice bounds out of range

goroutine 1 [running]:
github.com/tendermint/ethermint/vendor/github.com/ethereum/go-ethereum/cmd/utils.NewApp(0x4a48029, 0x5, 0x4a6666a, 0x24, 0xc4201cf168)
	/private/tmp/ethermint-20180124-17665-1kftk1c/ethermint-0.5.4/src/github.com/tendermint/ethermint/vendor/github.com/ethereum/go-ethereum/cmd/utils/flags.go:98 +0x265
main.init()
	/private/tmp/ethermint-20180124-17665-1kftk1c/ethermint-0.5.4/src/github.com/tendermint/ethermint/cmd/ethermint/main.go:18 +0x17a
Error: ethermint: failed
<0> expected but was
<2>.
```

We will need a test that does more than simply getting a version number.

Reply:
Something like starting a local node and checking it's status using curl?